Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -183,6 +192,31 @@ extension BrowserViewController: WKNavigationDelegate {
                 }
             }
     }
+    
+    private func displayErrorPage(error: NSError) {
+        if let webView = tabManager.selectedTab?.webView {
+            //TODO: I prefer to read errorURL from NSError, but it's weird
+            webView.loadHTMLString(self.generateErrorPage(error), baseURL: toolbar.currentURL())
+        }
+    }
+    
+    private func generateErrorPage(error: NSError) -> String {
+        
+        var resultString = ""
+        
+        resultString += "code: \(error.code)\n"
+        resultString += "domain: \(error.domain)\n"
+        resultString += "userInfo: \(error.userInfo)\n"
+        resultString += "localizedDescription: \(error.localizedDescription)\n"
+        resultString += "localizedRecoveryOptions: \(error.localizedRecoveryOptions)\n"
+        resultString += "localizedRecoverySuggestion: \(error.localizedRecoverySuggestion)\n"
+        resultString += "localizedFailureReason: \(error.localizedFailureReason)\n"
+        resultString += "recoveryAttempter: \(error.recoveryAttempter)\n"
+        resultString += "helpAnchor: \(error.helpAnchor)\n"
```

Comment:
I don't see `resultString` being used. Is this just for debugging or do you intent do include it on the page?

I think it would be interesting to display all those values on the page: that will give us a good idea of what is actually relevant and then we can decide what we want to show in a more final version.